metadata_backend/api/handlers.py

```diff
@@ -164,14 +164,16 @@ async def validate(self, req: Request) -> Response:
             raise web.HTTPBadRequest(reason=reason)
 
         except ParseError as error:
-            reason = f"Faulty XML file was given. Details: {error}"
-            body = json.dumps({"isValid": False, "reason": reason})
+            detail = f"Faulty XML file was given.<br/>ERROR: {error}"
```

Comment:
I would omit the `<br/>` from the response. it is not necessary to format JSON responses.

also in line `173` below we would not need to replace it.